.github/workflows/runbenchandupload.yml

```diff
@@ -0,0 +1,62 @@
+name: RunUploadBenchmarks
+env:
+  JULIA_NUM_THREADS: 1
+on:
+  pull_request:
+    types: [labeled]
+    branches:
+     - master
+     - benchx
+jobs:
+  benchmark:
+    if: ${{ github.event.label.name == 'to-benchmark' }}
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.allow_failure }}
+    permissions: write-all
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1'
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+        include:
+          - version: '1'
+            allow_failure: false
+    steps:
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      - uses: actions/cache@v3
+        env:
+          cache-name: cache-artifacts
+        with:
+          path: ~/.julia/artifacts
+          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
+          restore-keys: |
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
+      - uses: julia-actions/julia-buildpkg@latest
+      - name: install dependencies
+        run: julia -e 'using Pkg; pkg"add PkgBenchmark BenchmarkCI@0.1"'
+      - name: Run benchmark judge
+        run: julia -e "
+          using BenchmarkCI, PkgBenchmark; 
+          jd=BenchmarkCI.judge(baseline=\"origin/${GITHUB_BASE_REF}\"); 
+          "
+      - name: Save PR number
+        env:
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+        run: |
+          echo $PR_NUMBER > .benchmarkci/pr_number
+      - uses: actions/upload-artifact@v3
+        with: 
+          name: BenchmarkResults
+          path: .benchmarkci/
```

Comment:
so this is populated by `BenchmarkCI.judge`?

Reply:
yes, the content of `.benchmarkci` is generated by `BenchmarkCI.judge`.